forc-plugins/forc-tx/src/lib.rs

```diff
@@ -784,36 +790,41 @@ impl TryFrom<Input> for fuel_tx::Input {
                         err,
                     })?;
                 match (witness_ix, predicate.bytecode, predicate.data) {
-                    (Some(witness_index), None, None) => fuel_tx::Input::MessageSigned {
-                        message_id,
-                        sender,
-                        recipient,
-                        amount,
-                        nonce,
-                        data,
-                        witness_index,
-                    },
-                    (None, Some(predicate), Some(predicate_data)) => {
-                        fuel_tx::Input::MessagePredicate {
-                            message_id,
+                    (Some(witness_index), None, None) => fuel_tx::Input::MessageDataSigned(
```

Comment:
Looks like we now have `MessageCoinSigned` and `MessageDataSigned` introduced to `fuel-tx`. 

As far as I understood if data is present we should be creating `MessageDataSigned` as `MessageCoinSigned` needs an empty tuple for the data field. 

Should we make data optional and add it to match statement so that we also generate `MessageCoinSigned`?